projects/components/src/filtering/filter/parser/types/in-filter-parser.ts

```diff
@@ -40,7 +40,7 @@ export class InFilterParser extends AbstractFilterParser<PossibleValuesTypes> {
 
   private parseNumberArrayValue(valueString: string): number[] | undefined {
     const array = valueString.split(InFilterParser.IN_DELIMITER).map(str => {
-      const val = Number(str.trim());
+      const val = str.trim() === '' ? NaN : Number(str.trim());
```

Comment:
nit: we already have number coercer (although it may have this same bug, didn't realize javascript was so useful in converting an empty string to zero for us) - should we reuse that here?

Reply:
Just saw this comment hanging out down here at the bottom.

Yeah, our current coercer doesn't do this. Was tempted to to change it but wasn't bold enough to deal with any fallout at the moment.

Reply:
I don't think there should be any, but maybe we can do that in a separate PR